src/mudlet-lua/lua/Other.lua

```diff
@@ -692,6 +692,24 @@ function deleteFull()
   tempLineTrigger(1, 1, [[if isPrompt() then deleteLine() end]])
 end
 
+function deleteMultiline(maxLines)
+  local multimatchesSize = table.size(multimatches)
+  if multimatchesSize == 0 then
+    return nil, "Does not appear to be run during a multiline trigger match, please try again."
```

Comment:
Just a consistency thing:

```suggestion
    return nil, "does not appear to be run during a multiline trigger match, please try again"
```